### ISSUE TITLE
user report downloads contain virus

### ISSUE THREAD

**Issue by HinTak** [HinTak/Font-Validator#28]
Re-visit/investigate at some point in the future:

http://www.typophile.com/node/126470
from user Chris Hopkins

**Comment by Pomax** [comment]
I still recommend using the github release concept, rather than sourceforge, which _still_ has zero problems slipping "not your files" into their installers =)

**Comment by HinTak** [comment]
@Pomax Well, FontVal 2.1's binaries are on github, and will be from here simultaneously onwards - if there is a FontVal 2.2, that is. Sourceforge is a 17(?) year of fammiliarity despite its flaws, compared to me being on github for less than 2 years.

**Comment by Pomax** [comment]
Not as young as you might think - github's been around for almost 10 years now =)

**Comment by HinTak** [comment]
Yes, I have read up about the sourceforge issue. I haven't used it actively much lately and have not experienced any issue first hand, so good feelings from old times die hard... or does not die...

When I posted to typedrawers for 2.1, I put the github url first before the sourceforge one. The github download area is certainly easier for non-tech people. I'll slowly adjust... give me time, old dogs and new tricks, you know ;)

**Comment by Pomax** [comment]
yeah, it's a weird sourceforge quirk due to the large number of servers they use - not every server is "compromised" even when someone downloads an archive that clearly has a trojan or the like in it. It's a bit if Linux distribution locations didn't sync: every sourceforge server has its own copy of the files, but unlike linux distro locations, none of them do upstream sync/verification or even with each other, to make sure the payloads are identical and match the official hashes... so unless you're basically sitting next to your reporting user, the chances of getting the same file are almost zero =(
